src/tests/multihost/ipa/conftest.py

```diff
@@ -97,6 +138,28 @@ def remove_ipa_users():
     request.addfinalizer(remove_ipa_users)
 
 
+@pytest.fixture(scope="class")
+def default_ipa_groups(session_multihost, request):
+    """ Create IPA Groups ipa-group0 to ipa-group9 """
+    kinit_admin = 'kinit admin'
+    session_multihost.master[0].run_command(kinit_admin,
+                                            stdin_text='Secret123',
+                                            raiseonerr=False)
+    for i in range(10):
+        group_add = 'ipa group-add ipa-group%d --desc="IPA group%d"' % (i, i)
+
+        cmd = session_multihost.master[0].run_command(group_add,
```

Comment:
This command provides the resulting GID in the output.
I think the GID should be parsed from it.